microsoft_atp/plugin.spec.yaml

```diff
@@ -594,6 +722,100 @@ actions:
         description: List of machines with provided software
         type: '[]machine_software'
         required: true
+  blacklist:
+    title: Blacklist
+    description: Submit or update new indicator
+    input:
+      indicator:
+        title: Indicator
+        description: The value of the indicator
+        type: string
+        example: 220e7d15b011d7fac48f2bd61114db1022197f7f
+        required: true
+      indicator_type:
```

Comment:
We should auto-detect the indicator in the code so the user doesn't have to choose the type.

Reply:
@jschipp-r7 
If user leave empty this field, type will be autodetected so user doesn't have to choose it. We can remove this field of course if needed :)

Reply:
@r7-mgorny yeah, if it's auto-detected we don't need the extra input. Let's remove it - thanks